### PR TITLE
Address some tech debt

### PR DESCRIPTION
**Pull request by emmberk**

### CODE REVIEW

File: src/test/java/com/connexta/store/StoreControllerUnitTests.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  private StoreApi storeApi;
+
+  @BeforeEach
+  public void beforeEach() {
+    storeApi = new StoreController(mockStoreService, STORE_API_VERSION);
+  }
+
+  @ParameterizedTest
+  @NullAndEmptySource
+  @ValueSource(strings = "   ")
```
Comment (mcalcote) [inline src/test/java/com/connexta/store/StoreControllerUnitTests.java]: ❓ Are these parameterized now just so you can add more in the future?

Comment (mcalcote) [inline src/test/java/com/connexta/store/StoreControllerUnitTests.java]: ❓ Are these parameterized now just so you can add more in the future?

Comment (emmberk) [reply src/test/java/com/connexta/store/StoreControllerUnitTests.java]: No, I also test `NullAndEmptySource`.

File: src/main/java/com/connexta/store/controllers/MultipartFileValidator.java [hunk 2]
```diff
@@ -8,47 +8,34 @@
...
-      throw new ValidationException(s);
+  private static void validateNotBlank(String s, String message) {
+    if (isBlank(s)) {
+      throw new ValidationException(message);
     }
-    return mediaType;
   }
 
-  private void validateSize() {
+  private static void validateSize(final MultipartFile file) {
```
[the remark this answers is not in the document]
Comment (ahoffer) [reply src/main/java/com/connexta/store/controllers/MultipartFileValidator.java]: sounds good to me.

File: src/test/java/com/connexta/store/RetrieveFileTests.java [hunk 3]
```diff
@@ -54,10 +55,8 @@ public void testContextLoads() {}
   @ParameterizedTest(name = "400 Bad Request if ID is {0}")
   @ValueSource(
       strings = {"    ", "1234567890123456789012345678901234", "+0067360b70e4acfab561fe593ad3f7a"})
-  void testBadIds(String id) throws Exception {
-    mockMvc
-        .perform(MockMvcRequestBuilders.get("/dataset/{id}", id))
-        .andExpect(status().isBadRequest());
+  void testBadIds(final String datasetId) throws Exception {
```
[the remark this answers is not in the document]
Comment (ahoffer) [reply src/test/java/com/connexta/store/RetrieveFileTests.java]: Did those always work?

[the remark this answers is not in the document]
Comment (emmberk) [reply src/test/java/com/connexta/store/RetrieveFileTests.java]: Yep

[the remark this answers is not in the document]
Comment (ahoffer) [reply src/test/java/com/connexta/store/RetrieveFileTests.java]: Where is the scream emoji when I really need it?

[the remark this answers is not in the document]
Comment (emmberk) [reply src/test/java/com/connexta/store/RetrieveFileTests.java]: 😱

**Pull request merged**